### ISSUE TITLE
Simpler Roles (admin and premium user)

### ISSUE THREAD

**Issue by dinoboff** [singpath/classmentors#124]
Adding admin and premium user is cumbersome.

Instead of being a list and user profile attribute, it could simply be a list. And instead of a user id list, it should be public id list.

Alternatively we can leave it cumbersome for admins (there not that many admins to add) and add a form to add a premium users.

**Comment by dinoboff** [comment]
@SingaporeClouds what do you think?

Is it an issue? what would you favour?

**Comment by SingaporeClouds** [comment]
I do not think that it is important in the near future. I would prefer to see all the other issues closed before considering changes such as this.

**Comment by dinoboff** [comment]
Should I document how to add premium users with the current method?

https://github.com/singpath/classmentors/pull/120/files#diff-51c938448a46b23f62ff1bd8b021b350R130

**Comment by SingaporeClouds** [comment]
The setup doc should be the basics that are needed to get a copy of ClassMentors up and running quickly so that someone could run a short workshop or run a single activity in a class. Additional details such as how to add additional premium users can go in a different document somewhere along with other details that an admin might want to know. 

The priority is to first complete the "5-minutes to ClassMentors deployment" documentation.
